gluon/dal.py

```diff
@@ -117,3 +117,11 @@ def _default_validators(db, field):
         DRIVERS['pg8000'] = pg8000
     except:
         pass
+if not DRIVERS.get('sqlite3'):
```

Comment:
I will not merge a PR that contains a change to the web2py framework. 
is this a bug with dal or what?  we need to investigate.  If it's a bug with dal, we should report it to them, but I highly doubt it.  Let's discuss this in class.